src/routes/Answer.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
+	import { clone } from '$lib/transition/clone';
+	import { flip } from 'svelte/animate';
+	import { cubicInOut } from 'svelte/easing';
+	import { fade, slide } from 'svelte/transition';
+
+	export let values: (string | number)[] = [];
+	export let target: Element | null = null;
+
+	const options = { duration: 1400, easing: cubicInOut };
+</script>
+
+<div class="flex flex-col-reverse gap-2 min-h-60">
```

Comment:
I think flex-col might make more sense than flex-col-reverse? The user can input up to 99 zeroes, and I think it looks a little jank when the rest of the webpage jumps up to accommodate a new guess when the guess history area collides with the unit area. Aside from the jankiness it'll probably also cause more layout issues as we start filling out the page (adding questions, pictures, etc). With flex-col we can just pick a starting point that won't collide with the unit area and move down from there. There is something to be said for having your most recent guess at the top, but I think most -dle's have guesses grow down anyway -- it's what I'm used to at least. Let me know if you disagree!

Reply:
I don't think flex-col vs flex-col-reverse is the issue here, that just changes the order the guesses are laid out, they'll still take up the same amount of space. I think what you're referring to is more where the answers are justified, which apparently I didn't put in even though I thought I did? It's now in the latest commit

I thought it was more natural to have your latest guess be at the top (always in the same spot) but it doesn't have to be that way and tbh I'd have to play some more -dle's to get a good idea of how that should work.

I think the issue that you're seeing may be that we just need more space reserved for the number or for the guesses, I'm not exactly sure what the issue is though since I don't really see any jumping with a large number of 0's on my machine. I do see some jumping with a large number of guesses but that's only after they overflow their minimum height

Reply:
Looks a lot better now! One thing that's a bit weird is after you guess 6 times, any further guesses will update going from the bottom up. Obviously, we won't be allowing more than 6 guesses, so it shouldn't be a problem.